### ISSUE TITLE
Question about how a blood gas sample is defined as arterial

### ISSUE THREAD

**Issue by snowgato** [MIT-LCP/mimic-code#1161]
### Prerequisites

* [ X] Put an X between the brackets on this line if you have done all of the following:
    * Checked the online documentation: https://mimic.mit.edu/
    * Checked that your issue isn't already addressed: https://github.com/MIT-LCP/mimic-code/issues?utf8=%E2%9C%93&q=

### Description
Hello,
I get confused about how a blood gas sample is classified as an arterial blood gas sample in mimic iv database.
In mimic_derived, there's a script, bg.sql, that defines mimic_derived.bg table containing specimen field supposed to designate the nature of the blood gas sample, wether it is arterial or not.
This field is actually always null in mimic_derived.bg table.
According to bg.sql, specimen field corresponds to labevents records with itemid equal to 52028.
But strangely, the label associated to itemid in d_labitems is Hematocrit, and actually there's not a single labevents record with itemid equal to 52028. 

In bg.sql, 
specimen_pred is set to specimen if not null, 
else set to 'ART.' if specimen_prob>0,75 
else it is null.
As specimen is always null, 
finally specimen_pred is only set to 'ART.' if specimen_prob>0,75, else null.

Finally mimic_derived.first_day_bg_art is defined out of mimic_derived.bg table where specimen_pred is equal to 'ART.'
As a consequence, all recordings defined as arterial blood gas in first_day_bg_art actually rely on a probability computation specimen_prob and never on specimen tagged as 'ART.' before being sent to the laboratory for analysis.

If someone has an explanation about that, I'd greatly appreciate it.
Thanks

**Comment by alistairewj** [comment]
> Hello, I get confused about how a blood gas sample is classified as an arterial blood gas sample in mimic iv database. In mimic_derived, there's a script, bg.sql, that defines mimic_derived.bg table containing specimen field supposed to designate the nature of the blood gas sample, wether it is arterial or not. This field is actually always null in mimic_derived.bg table. According to bg.sql, specimen field corresponds to labevents records with itemid equal to 52028. But strangely, the label associated to itemid in d_labitems is Hematocrit, and actually there's not a single labevents record with itemid equal to 52028.

Yes, this is a bug. I retained the wrong `itemid` here and it should instead be 52033.

> In bg.sql, specimen_pred is set to specimen if not null, else set to 'ART.' if specimen_prob>0,75 else it is null. As specimen is always null, finally specimen_pred is only set to 'ART.' if specimen_prob>0,75, else null.

This is based off of a regression model I built to impute the specimen type if missing.

> Finally mimic_derived.first_day_bg_art is defined out of mimic_derived.bg table where specimen_pred is equal to 'ART.' As a consequence, all recordings defined as arterial blood gas in first_day_bg_art actually rely on a probability computation specimen_prob and never on specimen tagged as 'ART.' before being sent to the laboratory for analysis.
> 
> If someone has an explanation about that, I'd greatly appreciate it. Thanks

This is definitely unideal! And in fact, I don't think the model is needed anymore, as we should always have the specimen now as we improved data collection in MIMIC-IV.

**Comment by alistairewj** [comment]
See also #923 .. kinda slow on the fix, sorry.

**Comment by alistairewj** [comment]
Comparison of the model's predictions with the ground truth:

Row | specimen_pred | specimen | count
--- | --- | --- | ---
1 | null | null | 14909 |  
2 | null | ART. | 9169 |  
3 | null | CENTRAL VENOUS. | 4479 |  
4 | null | GV. | 3 |  
5 | null | MIX. | 6803 |  
6 | null | VEN. | 41833 |  
7 | ART. | null | 14874 |  
8 | ART. | ART. | 411402 |  
9 | ART. | CENTRAL VENOUS. | 2993 |  
10 | ART. | GV. | 2 |  
11 | ART. | MIX. | 4043 |  
12 | ART. | VEN. | 50702 |

Unfortunately still ~30,000 specimens without a label. I'll still remove the model and let individuals handle the missing data as they prefer.

**Comment by snowgato** [comment]
Hi,
Thanks for your answer and bug fix.
Have you modified accordingly first_day_bg_art in mimic_derived ?
